ansible_rulebook/engine.py

```diff
@@ -373,7 +377,9 @@ async def call_action(
                     action_args["project_data_file"] = self.project_data_file
                     action_args["source_ruleset_name"] = ruleset
                     action_args["source_rule_name"] = rule
-                    return await self.pa_runner.wait_for_playbook(action_args)
+                    return await self.pa_runner.wait_for_playbook(
```

Comment:
can we rename `wait_for_playbook`? now it's not for playbook only

Reply:
I added some comments to explain the whole class is for both playbook and module.